FordLabs/PeopleMover
# Fix last updated timestamp

## Issue
Connects #353 

## What was done
- [x] Fixed the backend so that the last updated timestamp updates to the current time each time it is used.

## How to test
1. Go to a space and ensure that the updated date gets updated for each of the below instances:
- Add/Edit/Delete Product
- Add/Delete/Edit Person
- Add/Edit/Delete Role
- Edit/Delete Product Tag (NOT Add)
- Add/Edit/Delete Location
- Add/Cancel Assignments
- Add Space (NOT Edit)
